src/sentry/static/sentry/app/styles/pulsingIndicator.tsx

```diff
@@ -0,0 +1,39 @@
+import {css, keyframes} from '@emotion/core';
+
+import {Theme} from 'app/utils/theme';
+
+const pulse = keyframes`
+  0% {
+    transform: scale(0.1);
+    opacity: 1
+  }
+
+  40%, 100% {
+    transform: scale(0.8);
+    opacity: 0;
+  }
+`;
+
+const pulsingIndicatorStyles = (p: {theme: Theme}) => css`
```

Comment:
Probably another scenario where I can make this into a component?

Reply:
Hmm maybe, it doesn't seem as "generic" as the Card